.github/workflows/publish_helm_charts.yml

```diff
@@ -0,0 +1,43 @@
+# this is WIP and needs a revision and testing before merging. Also we need to add this secrets in GH secrets
+name: Publish helm charts
+on:
+# Will only run when release is published. 
+  push:
```

Comment:
The triggers seem off for a number of the jobs - for example, this means run on any push to any branch right?

Reply:
ignore triggers for now. I have added those so I can see those workflows spinning right now. Will change triggers while converting it to PR from draft.